clickhouse_search/fixtures/clickhouse_search.json

```diff
@@ -222,10 +222,22 @@
       "sorted_regulatory_feature_consequences": []
     }
 }, {
-    "model": "clickhouse_search.clinvarsnvindel",
-    "pk": 1,
+    "model": "clickhouse_search.keylookupsnvindel",
+    "pk": "1-10439-AC-A",
     "fields": {
-      "key": 1,
```

Comment:
rather than directly insert fixture rows into the clinvar join table, insert into the all variants tables and rely on the materialized view to work correctly. Changing the data like this ensures that out unit tests will catch if those views stop working as intended